odoo/orm/models.py

```diff
@@ -147,12 +147,13 @@ def fix_import_export_id_paths(fieldname):
     return fixed_external_id.split('/')
 
 
-def to_company_ids(companies):
+def to_model_ids(companies):
```

Comment:
If you rename the method, please give it a better name and overall API.
```suggestion
def to_record_ids(arg) -> list[int]:
    """ Return the record ids of ``arg``, which may be a recordset, an integer or a list of integers. """
```